lib/ckb/api.rb

```diff
@@ -152,7 +152,7 @@ def get_live_cell(out_point, with_data = false)
     # @param transaction [CKB::Types::Transaction]
     #
     # @return [String] tx_hash
-    def send_transaction(transaction, outputs_validator = "default")
+    def send_transaction(transaction, outputs_validator = "passthrough")
```

Comment:
Why set the `passthrough` as the default value?

Reply:
> Null outputs_validator means passthrough
null is the default value on ckb, so I change `default` to `passthrough`

Reply:
> Why set the `passthrough` as the default value?

after thinking about it, I will change it to null by default. thanks.